src/main/java/org/xstefank/model/Utils.java

```diff
@@ -62,4 +63,12 @@ public static URL getFormatUrl() throws MalformedURLException {
             target = TyrProperties.getProperty(TEMPLATE_FORMAT_URL);
         return target != null ? new URL(target) : null;
     }
+
+    private static String getConfigDirectory() {
+        String path = System.getProperty(TYR_CONFIG_DIR);
+        if (path == null) {
+            throw new IllegalStateException("Configuration directory property is not set");
```

Comment:
I just wonder if this needs to be required, can we take for default current directory for instance?

Reply:
I think that we never know which application server would be used and I think every server has different property. What comes to my mind is that we could have maybe 3-4 different application servers properties that we could try to parse and if all of them will be null and also that `TYR_CONFIG_DIR` will be null it could throw an exception. What do you think about that?